tests/MoBi.Tests/Core/SBML/UnitImporterSpecs.cs

```diff
@@ -117,7 +100,7 @@ protected override void Context()
       [Observation]
       public void ParameterWithVolumeUnitTest()
       {
-         var tc = _moBiProject.SpatialStructureCollection.FirstOrDefault().TopContainers.FirstOrDefault();
+         var tc = _moBiProject.SpatialStructureCollection.First().TopContainers.First();
```

Comment:
It should always be First since you are calling something after. FirstOrDefault does not make sense. Probably old code but we might as well fix it

Reply:
Sure, you are right.